[3/1709] CXX obj/third_party/blink/renderer/platform/platform/platform_jumbo_6.o
  ./../../../src/third_party/blink/renderer/platform/graphics/gpu/dawn_control_client_holder.cc:161:7: error: no member named 'kEnableUnsafeWebGPU' in namespace 'blink::switches'; did you mean '::switches::kEnableUnsafeWebGPU'?
    161 |       switches::kEnableUnsafeWebGPU);
        |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        |       ::switches::kEnableUnsafeWebGPU
  ../../../src/gpu/config/gpu_switches.h:21:30: note: '::switches::kEnableUnsafeWebGPU' declared here
    21 | GPU_EXPORT extern const char kEnableUnsafeWebGPU[];
        |                              ^
  In file included from gen/third_party/blink/renderer/platform/platform_jumbo_6.cc:85:
  ./../../../src/third_party/blink/renderer/platform/graphics/gpu/dawn_control_client_holder.cc:175:27: error: no member named 'kWGSLUnsafeFeatures' in namespace 'blink::features'; did you mean '::features::kWGSLUnsafeFeatures'?
    175 |         base::SplitString(features::kWGSLUnsafeFeatures.Get(), ",",
        |                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        |                           ::features::kWGSLUnsafeFeatures
  ../../../src/gpu/config/gpu_finch_features.h:123:57: note: '::features::kWGSLUnsafeFeatures' declared here
    123 | GPU_EXPORT extern const base::FeatureParam<std::string> kWGSLUnsafeFeatures;
        |                                                         ^
  2 errors generated.
--- a/third_party/blink/renderer/platform/graphics/gpu/dawn_control_client_holder.cc
+++ b/third_party/blink/renderer/platform/graphics/gpu/dawn_control_client_holder.cc
@@ -158,7 +158,7 @@
   WGPUDawnWireWGSLControl wgsl_control = {};
   wgsl_control.chain.sType = WGPUSType_DawnWireWGSLControl;
   wgsl_control.enableUnsafe = base::CommandLine::ForCurrentProcess()->HasSwitch(
-      switches::kEnableUnsafeWebGPU);
+      ::switches::kEnableUnsafeWebGPU);
   wgsl_control.enableExperimental =
       wgsl_control.enableUnsafe ||
       RuntimeEnabledFeatures::WebGPUDeveloperFeaturesEnabled();
@@ -172,7 +172,7 @@
 
   if (!wgsl_control.enableUnsafe) {
     wgsl_unsafe_features_owned =
-        base::SplitString(features::kWGSLUnsafeFeatures.Get(), ",",
+        base::SplitString(::features::kWGSLUnsafeFeatures.Get(), ",",
                           base::TRIM_WHITESPACE, base::SPLIT_WANT_ALL);
     wgsl_unsafe_features.reserve(wgsl_unsafe_features_owned.size());
     for (const auto& f : wgsl_unsafe_features_owned) {

[3/1708] CXX obj/third_party/blink/renderer/platform/platform/platform_jumbo_8.o
  ./../../../src/third_party/blink/renderer/platform/graphics/video_frame_submitter.cc:345:7: error: no member named 'IsOnBeginFrameAcksEnabled' in namespace 'blink::features'; did you mean '::features::IsOnBeginFrameAcksEnabled'?
    345 |   if (features::IsOnBeginFrameAcksEnabled()) {
        |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        |       ::features::IsOnBeginFrameAcksEnabled
  ../../../src/components/viz/common/features.h:126:24: note: '::features::IsOnBeginFrameAcksEnabled' declared here
    126 | VIZ_COMMON_EXPORT bool IsOnBeginFrameAcksEnabled();
        |                        ^
  1 error generated.
--- a/third_party/blink/renderer/platform/graphics/video_frame_submitter.cc
+++ b/third_party/blink/renderer/platform/graphics/video_frame_submitter.cc
@@ -342,7 +342,7 @@
     const WTF::HashMap<uint32_t, viz::FrameTimingDetails>& timing_details,
     bool frame_ack,
     WTF::Vector<viz::ReturnedResource> resources) {
-  if (features::IsOnBeginFrameAcksEnabled()) {
+  if (::features::IsOnBeginFrameAcksEnabled()) {
     if (frame_ack) {
       DidReceiveCompositorFrameAck(std::move(resources));
     } else if (!resources.empty()) {
